cfme/containers/image.py

```diff
@@ -59,6 +59,43 @@ def perform_smartstate_analysis(self, wait_for_finish=False):
                 raise TimedOutError('Timeout exceeded, Waited too much time for SSA to finish ({}).'
                                     .format(ssa_timeout))
 
+    def check_compliance(self, timeout=240):
+        """Initiates compliance check and waits for it to finish."""
+        original_state = self.compliance_status
+        tb.select('Policy',
+                  "Check Compliance of Last Known Configuration",
+                  invokes_alert=True)
+        sel.handle_alert()
+        flash.assert_no_errors()
+        wait_for(
+            lambda: self.compliance_status != original_state,
+            num_sec=timeout, delay=5, fail_func=sel.refresh,
+            message="compliance of {} checked".format(self.name)
+        )
+        return self.compliant
+
+    @property
+    def compliance_status(self):
+        self.summary.reload()
```

Comment:
yes. summary reload [is calling to self.load_details](https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/__init__.py#L211), which [is doing the navigation](https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/image.py#L33).